lib/ci/run_ci.js

```diff
@@ -71,6 +76,28 @@ export class RunPRJob {
         return false;
       }
       cli.stopSpinner('PR CI job successfully started');
+
+      // check if the job need a v8 build and trigger it
+      await this.prData.getPR();
+      const labels = this.prData.pr.labels
+      if (labels.includes('v8 engine')) {
+        cli.startSpinner('Starting V8 CI job');
+        const response = await this.request.fetch(CI_V8_URL, {
+          method: 'POST',
+          headers: {
+            'Jenkins-Crumb': crumb
+          },
+          body: this.payload
```

Comment:
This won't work for the V8 CI job as it takes different parameters to the pull request job.